internal/errors/errors.go

```diff
@@ -0,0 +1,7 @@
+package errors
+
+import "errors"
+
+var (
+	ErrInvalidReference = errors.New("image reference format is invalid. Please specify <name:tag|name@digest>")
```

Comment:
The error message is too general. For `oras copy`, how do we know this error message is for the source or the destination?

Reply:
Let me add registry and repository in so user can have more context on the cause of the error